### DISCUSSION TITLE
zh-cn/manual/introduction/reclamation-algorithm

### DISCUSSION THREAD

**Discussion by giscus** [MaaAssistantArknights/maa-website#172] [Comments]
# zh-cn/manual/introduction/reclamation-algorithm

生息演算 目前生息演算的支持仍处于早期阶段，暂不推荐无人值守，现阶段无法保证稳定性。 默认模式 需要在生息演算主界面开始任务（导航还没写） 不能在已经有存档的情况下使用（删除现有存档即可） 不能在生息演算的编队中有干员的情况下使用（把生息演算中的当前编队清空即可，特别是打完浪潮之后） 制造刷点数 需要在进入生息演算中能看到驻扎地的页面开始 若未填写道具...

https://docs.maa.plus/zh-cn/manual/introduction/reclamation-algorithm.html

**Comment by inrentmoon** [comment]
为什么生稀盐酸经常运行到一半就显示任务出错，一般在第一天地图界面或者关卡里

**Reply by jadaprior** [reply]
同问，请问你后来解决了吗？

**Comment by whdtx111** [comment]
进关卡的时候，确认无干员配对的情况下进入，一直卡提示，需要手动点击确定才能继续

**Comment by X-Xyy** [comment]
RA4的时候维什戴尔放不下去，PC端

**Reply by shyu216** [reply]
我雷电模拟器原来1280，也放不下去，改成1920就好了。你试试看？

**Comment by hai1214642026-dev** [comment]
生息演算第二期RA4经常会刷一会就卡在助战界面或者在局内战斗界面

**Comment by chenanzhi0033-dotcom** [comment]
生息演算-重启锚点-RA-1，一直会卡住，基地和干员放不下去

**Reply by ABA2396** [reply]
按提示做了没

**Comment by Yyrx5555** [comment]
用maa在PC端刷重生锚点RA-15的时候，鼠标点击的地方一直在采集基站上面一点，也没法放置干员，不知道是不是我的设置的问题

**Reply by Yyrx5555** [reply]
噢噢噢，换成SendMsg就好了，用前台Seize就总是差点位置
